lib/block-supports/layout.php

```diff
@@ -34,10 +34,11 @@ function gutenberg_register_layout_support( $block_type ) {
  * @param string  $gap_value                     The block gap value to apply.
  * @param boolean $should_skip_gap_serialization Whether to skip applying the user-defined value set in the editor.
  * @param string  $fallback_gap_value            The block gap value to apply.
+ * @param array   $block_padding                 Custom padding set on the block.
  *
  * @return string                                CSS style.
  */
-function gutenberg_get_layout_style( $selector, $layout, $has_block_gap_support = false, $gap_value = null, $should_skip_gap_serialization = false, $fallback_gap_value = '0.5em' ) {
+function gutenberg_get_layout_style( $selector, $layout, $has_block_gap_support = false, $gap_value = null, $should_skip_gap_serialization = false, $fallback_gap_value = '0.5em', $block_padding = null ) {
```

Comment:
Oh, just another thought — since we're adding an additional param, would it be worth changing `$block_padding` for `$block_spacing` in case we ever need to add width, height or other properties, or handle margin values that are set on the block?

Reply:
And passing in the whole 'spacing' array instead of just the 'padding' part of it? Yeah, I guess so!

Reply:
Yep, that's what I was thinking 🙂